functions/src/monitor/monitor.ts

```diff
@@ -18,9 +20,10 @@ export const getMonitorPings: any = async (projectUid: string, monitorUid: strin
     .collection('projects')
     .doc(projectUid)
     .get();
-  const project: ProjectModel = <ProjectModel> document.data();
+  const project: ProjectModel = <ProjectModel>document.data();
```

Comment:
When type casting its easier to read with a space

```typescript
<ProjectModel> document.data();
```

Reply:
Done